examples/luamtev/hello-world.lua

```diff
@@ -0,0 +1,113 @@
+-- luamtev applications need to be wrapped in a module
+module(...,package.seeall)
+
+-- cache pid/tid
+local pid, tid = mtev.thread_self()
+
+-- initialize RNG
+local s,us = mtev.gettimeofday()
+math.randomseed(us)
+
+function coro_main(coro_id, w)
+  -- Q: Is there any way to get the identity of the currently running coroutine?
+  --    like mtev.thread_self() just for co-routines?
+  local wait = math.random(w)
+  mtev.log("out","Hello from coro %d! waiting %d\n", coro_id, wait)
+
+  -- suspend this co-routine. Yield to event loop.
+  local slept = mtev.sleep(wait)
+  mtev.notify("SLEEP", coro_id, slept:seconds())
+
+  -- do some async I/O from the co-routine
+  local proc = mtev.spawn("/bin/echo", { "echo", "Hello from echo" })
+  local status, errno = proc:wait(1)
+  mtev.notify("PROC", coro_id)
+  -- Q: How can we communicate with the subprocess?
+  --    - read from stdout/stderr
+  --    - write to stdin
+
+  -- TODO: Add HTTP GET example
+
+  -- tell main() we are done
+  mtev.notify("DONE", coro_id)
+end
+
+--
+-- The main function is executed concurrently by all lua threads
+-- when <concurrent>true</concurrent> is set in the config
+--
+-- Q: How to configure the threaded concurrency?
+-- Apparently <default_queue_threads>10</default_queue_threads> in eventer/config is not doing it.
```

Comment:
It's is the event loop concurrency not the jobq concurrency.

Reply:
Which config stanza would that require?